.gitignore

```diff
@@ -1,2 +1 @@
-bin/
-obj/
+/artifacts/
```

Comment:
why is bin/ and obj/ not needed anymore?

Reply:
With `ArtifactsPath` set, all the build output is in the central `/artifacts` directory